packages/frontend/src/components/Notifications/NotificationToast.tsx

```diff
@@ -0,0 +1,94 @@
+import * as ToastPrimitive from '@radix-ui/react-toast'
+import { useCallback } from 'react'
+import { CrossIcon } from 'src/components/Icons/CrossIcon'
+import { Colors } from 'src/styles/colors'
+import styled from 'styled-components'
+
+import { ErrorIcon } from '../Icons/ErrorIcon'
+
+interface Props {
+  message: string
+  setError: (str?: string) => void
+}
+
+export const NotificationToast = ({ message, setError }: Props) => {
+  const removeNotification = useCallback(() => setError(undefined), [setError])
+
+  return (
+    <Toast onOpenChange={(open) => open || removeNotification()} duration={5000}>
+      <NotificationIconWrapper>
+        <ErrorIcon color={Colors.Red} size={24} />
+      </NotificationIconWrapper>
+      <ToastContent>
+        <NotificationTitle>Error</NotificationTitle>
+        <NotificationDescription>{message}</NotificationDescription>
+        <NotificationActionText>Try Again</NotificationActionText>
```

Comment:
`Try Again` text looks like it can be clicked and it would retry failed action. I see that was designed like that so we can make that clickable and retry failed action.